### ISSUE TITLE
New feature: add fallback for the 'not found' page.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#4302]
Thinking about expanding #4293, I propose we support the folling structure in `config.yml`: 

```
notfound: [ 'page/not-found', 'page-404.twig' ]
```

This would mean Bolt would first try to render `page/not-found`, and if that doesn't exist, it'll fall back to static rendering of `page-404.twig`.

**Comment by GawainLynch** [comment]
As per IRC… This totally has my vote

**Comment by Boorj** [comment]
Please, don't forget haml (and jade, and others?) :pray:

**Comment by SahAssar** [comment]
@bobdenotter: Is this something we still plan for a 3,2? Or later?

**Comment by bobdenotter** [comment]
Nah, let's close this one for now. If somebody needs this, we can revisit.

**Comment by Boorj** [comment]
Is it implemented? or other way to do that?
